sdk/appcenter/src/main/java/com/microsoft/appcenter/Constants.java

```diff
@@ -31,7 +31,7 @@
     /**
      * Maximum allowed length for user identifier for App Center server.
      */
-    static final int USER_ID_APP_CENTER_MAX_LENGTH = 256;
+    public static final int USER_ID_APP_CENTER_MAX_LENGTH = 256;
```

Comment:
We should move to UserIdUtils

Reply:
It's used in `AppCenterUserIdTest` though to generate long user ids